Documentation/ceph-object-bucket-claim.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Object Bucket Claim CRD
+weight: 2800
+indent: true
+---
+
+# Ceph Object Bucket Claim
+
+Rook supports the creation of new buckets (and access to existing buckets) via two custom resources:
+
+- an `Object Bucket Claim (OBC)` is custom resource which requests a bucket (new or existing) and is described by a Custom Resource Definition (CRD) shown below.
+- an `Object Bucket (OB)` is a custom resource generated when a bucket is provisioned and is described by an OB CRD, also shown below.
+
+An OBC references a storage class which is created by an administrator. The storage class defines whether the bucket requested is a new bucket or an existing bucket. It also defined the bucket retention policy.
+
+## Sample
```

Comment:
I prefer the sample order in the lib [design doc](https://github.com/kube-object-storage/lib-bucket-provisioner/blob/master/doc/design/object-bucket-lib.md#api-specifications):
 - obc
 - secret and cm
 - app pod
 - ob
 - ob and obc CRDs

Reply:
I don't expect the storage class example to appear first since a SC, though important, is not the focus here. See my comment above.